lms/djangoapps/instructor_task/models.py

```diff
@@ -101,6 +103,17 @@ def create(cls, course_id, task_type, task_key, task_input, requester):
         task_id = str(uuid4())
         json_task_input = json.dumps(task_input)
 
+        # check length of task_input, and return an exception if it's too long
+        if len(json_task_input) > TASK_INPUT_LENGTH:
+            logger.error(
+                u'Task input longer than: `%s` for `%s` of course: `%s`',
```

Comment:
Since the save is not possible for large length, we can log the length for which the task has failed. It sounds like an edge case but if it fails for any reason in the future, we can compare the difference. Just a suggestion.

Reply:
@DawoudSheraz I am just trying to avoid an additional declaration to store the length of task input.

Reply:
We don't have to create a new variable to store the length. We can utilize `json_task_input`. Additionally, I think `len(json_task_input)` has a nuance cost in terms of performance.

Reply:
Yes, we are doing `len(json_task_input)` in above line. I don't think it is worth the cost and if it would be of any help to know the length of the given input as far as we know the allowed length.
Please let me know if you think otherwise.